tests/unit/providers/test_portkey_provider.py

```
@@ -1,7 +1,116 @@
+                "model": "test-model",
+                "choices": [
+                    {
+                        "index": 0,
+                        "message": {"role": "assistant", "content": "<think>because</think>answer"},
+                        "finish_reason": "stop",
+                    }
+                ],
+            }
+        )
+
+    async def list(self, **_kwargs: Any) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_kwargs`

(ANN401)